src/tools/clippy/clippy_lints/src/missing_const_for_fn.rs

```diff
@@ -156,7 +156,7 @@ impl<'tcx> LateLintPass<'tcx> for MissingConstForFn {
 
         let mir = cx.tcx.optimized_mir(def_id);
 
-        if let Ok(()) = is_min_const_fn(cx.tcx, mir, &self.msrv)
+        if let Ok(()) = is_min_const_fn(cx.tcx, &mir, &self.msrv)
```

Comment:
`mir` should already be a reference here, so I don't think this diff is required, right?

Reply:
Done, you're right!